_data/communication.yml

```diff
@@ -34,6 +34,12 @@ websites:
       twitter: CarbonBlack_Inc
       facebook: carbonblackinc
 
+    - name: Cloze
+      url: https://www.cloze.com/
+      img: cloze.png
+      tfa: Yes
+      doc: https://help.cloze.com/knowledge_base/topics/how-do-i-enable-2-step-authentication
```

Comment:
```suggestion
      doc: https://help.cloze.com/knowledge_base/topics/how-do-i-enable-2-step-authentication
      sms: Yes
```

Reply:
@vc7ee I've now added sms and software (as per the new documentation) to the suggested change, would you please remove this suggestion?